src/renderer/components/depositActions/forms/Leave.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback } from 'react'
+
+import { Form } from 'antd'
+import { useIntl } from 'react-intl'
+
+import { Input } from '../../uielements/input'
+import * as Styled from './Forms.styles'
+
+const getLeavedMemo = (thorAddress: string) => `LEAVE:${thorAddress}`
+
+const memoPlaceholder = getLeavedMemo('THORADDRESS')
+
+type FormValues = { memo: string }
+
+type Props = {
+  onFinish: (leaveData: { memo: string }) => void
+}
+export const Leave: React.FC<Props> = ({ onFinish: onFinishProp }) => {
+  const intl = useIntl()
+  const [form] = Form.useForm<FormValues>()
+
+  const onFinish = useCallback(
+    ({ memo }: FormValues) => {
+      onFinishProp({
+        memo
+      })
+    },
+    [onFinishProp]
+  )
+
+  return (
+    <Styled.Form
+      form={form}
+      /**
+       * typescast here caused by TS can not infer "any" type from onFinish cb and gives unknown here
+       */
+      onFinish={(data) => onFinish(data as FormValues)}
+      initialValues={{ memo: memoPlaceholder }}>
+      <div>
+        <Styled.InputContainer>
+          <Styled.InputLabel>memo</Styled.InputLabel>
+          <Styled.Form.Item
```

Comment:
In case of using `Innerform` in * .styles, just use antd `<Form.Item ...` here and at other places.